### PR TITLE
feat(pfFilter): Add filter options - inline, results count, items label

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
## Description

Provides options to:

- display filter and result on a single line
- show the total count in the results label (i.e. 5 of 10 Results)
- change the label from Results to a passed in string

## PR Checklist

- [x] Unit tests are included
- [x] Screenshots are attached (if there are visual changes in the UI)
- [x] A Designer (@beanh66) is assigned as a reviewer (if there are visual changes in the UI)
- [x] A CSS rep (@cshinn) is assigned as a reviewer (if there are visual changes in the UI)

## Rawgit Link (in lieu of screen shots)
https://rawgit.com/jeff-phillips-18/angular-patternfly/filter-docs/dist/docs/index.html#/api/patternfly.filters.component:pfFilter

### CODE REVIEW

File: styles/angular-patternfly.less [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "../node_modules/patternfly/node_modules/bootstrap/less/variables.less";
```
Comment (dtaylor113) [inline styles/angular-patternfly.less]: What?!  Didn't you stop me from adding bootstrap variables?  #543

Comment (jeff-phillips-18) [reply styles/angular-patternfly.less]: LOL, right... I will remove.

File: test/filters/filter.spec.js [hunk 2]
```diff
@@ -207,4 +207,51 @@ describe('Directive:  pfFilter', function () {
 
     expect(element.find('.pf-table-view-selected-label').text()).toContain('0 of 10 selected');
   });
+
+  it('should show results inline when specified', function() {
```
Comment (dtaylor113) [inline test/filters/filter.spec.js]: To be truly well rounded, each of these three tests need a negative test.  Ex:  "should not show results inline when not specified"  ```expect(element.find('.filter-pf.inline-filter-pf').length).toBe(0);```

File: src/filters/filters.less [hunk 3]
```diff
@@ -98,3 +98,55 @@ pf-filter-panel {
...
+    margin: (@grid-gutter-width / 2) (@grid-gutter-width / 2) (@grid-gutter-width / 4) 0;
+
+    pf-filter-fields,
+    pf-filter-results {
+      display: inline-block;
+    }
+
+    .form-group {
+      margin-bottom: 0;
+      @media (min-width: 345px) {
```
Comment (dtaylor113) [inline src/filters/filters.less]: Is there a variable for this?  Thought this would be why you @imported bootstrap variables.less?

**Comment by dtaylor113** [comment]
Other than the minor issues I raised, LGTM

File: src/filters/filters.less (hunk 3 above)
Comment (dtaylor113) [reply src/filters/filters.less]: Are there no patternfly variables which denote the min-widths at diff screen sizes?  If not, we should file an issue and add our own, then we don't have to worry about bootstrap variables being overwritten.

Comment (jeff-phillips-18) [reply src/filters/filters.less]: Removing as this is unnecessary. Thanks.

File: test/filters/filter.spec.js (hunk 2 above)
Comment (jeff-phillips-18) [reply test/filters/filter.spec.js]: Added negative tests

Comment (dtaylor113) [reply test/filters/filter.spec.js]: You did?  I don't see them?

**Comment by dtaylor113** [comment]
@cdcabrera, ok if I merge this or do you want a chance to review?  -thanks

**Comment by cdcabrera** [comment]
@dtaylor113 taking a look

**Pull request merged**